@xen-orchestra/web-core/lib/components/space-card/VtsSpaceCard.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <UiCard class="card-container">
+    <UiCardTitle>
+      {{ t('space') }}
+    </UiCardTitle>
+
+    <div class="content">
+      <VtsProgressBar noruler :current="used" :total="total" :label="name" legend-type="percent" class="progress" />
+
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('used-space') }}</template>
+        <template #value> {{ usedSpace.formattedValue }} </template>
```

Comment:
Be careful with unwanted spaces, sometimes it can cause display issues:
```suggestion
        <template #value>{{ usedSpace.formattedValue }}</template>
```